bundles/org.openhab.binding.tesla/src/main/java/org/openhab/binding/tesla/internal/handler/TeslaVehicleHandler.java

```diff
@@ -1027,8 +1027,7 @@ protected boolean establishEventStream() {
                 if (!isEstablished) {
                     eventBufferedReader = null;
 
-                    eventClient = clientBuilder.build()
-                            .register(new Authenticator((String) getConfig().get(CONFIG_USERNAME), vehicle.tokens[0]));
+                    eventClient = clientBuilder.build().register(new Authenticator("", vehicle.tokens[0]));
```

Comment:
It seems that this is the only usage of `Authenticator`, so `user` could be safely removed from the constructor. However, in either case please also check this method:

```
        private String getBasicAuthentication() {
            String token = this.user + ":" + this.password;
            return "Basic " + Base64.getEncoder().encodeToString(token.getBytes(StandardCharsets.UTF_8));
        }
```

It seems like this could also be simplified, or actually might need to be rewritten to some extend, if class would now only host a password (or token)?

Reply:
Haha, you caught me there @jlaur!
I was actually just at this piece of code last night, when I wondered why it still uses the credentials and then I became too tired and went to bed.
I had forgotten about it today and was just happy that it compiled (because I had put in "" as a test last night). 🤣 
Looking at it now again, I noticed that this is very old code for event streaming, which didn't work since years anymore - I hence now completely removed these parts.